### PR TITLE
New homepage

### PR DESCRIPTION
**Pull request by RoyEJohnson**
Issue #2106

### CODE REVIEW

File: src/app/pages/home/tutor/tutor.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            <a className="btn primary" href={data.buttonLink}>{data.buttonText}</a>
+            <a className="btn" href={data.demoLink}>{data.demoText}</a>
+        </div>
+    );
+}
+
+function IconGroup({icon, text}) {
+    return (
+        <div className="icon-group">
+            <img src={icon} alt />
```
Comment (nathanstitt) [inline src/app/pages/home/tutor/tutor.js:16]: should `alt` have a value?  Seems strange to have an empty attribute like this

**Review by nathanstitt — APPROVED** [review]
looks good Roy, a few small comments only

File: src/app/components/shell/shell.js [hunk 2]
```diff
@@ -32,16 +28,34 @@ function setUpBus(mainEl) {
...
+    const [Content, setContent] = React.useState(Suspense);
+
+    React.useEffect(
+        () => import(`./${name}/${name}`)
+            .then((content) => setContent(<content.default />)),
+        [name]
+    );
+
+    return Content;
+}
```
[the remark this answers is not in the document]
Comment (RoyEJohnson) [reply src/app/components/shell/shell.js:48]: Good to know. I'll revisit this in the future.

File: src/app/pages/home/tutor/tutor.js (hunk 1 above)
Comment (RoyEJohnson) [reply src/app/pages/home/tutor/tutor.js:16]: Images that don't contribute meaning to the page are supposed to have empty alts so the screenreader doesn't include them. I've gone back and forth between `alt` and `alt=""`

Comment (nathanstitt) [reply src/app/pages/home/tutor/tutor.js:16]: oh cool, I did not know that.  Good to learn!

**Pull request merged**